src/Illuminate/Database/Console/DatabaseInspectionCommand.php

```diff
@@ -19,13 +19,15 @@ abstract class DatabaseInspectionCommand extends Command
      * @param  \Illuminate\Database\ConnectionInterface  $connection
      * @param  string  $database
      * @return string
+     *
+     * @deprecated Use $connection->getName() instead.
      */
     protected function getConnectionName(ConnectionInterface $connection, $database)
     {
         return match (true) {
+            $connection instanceof MariaDbConnection => 'MariaDB',
```

Comment:
Bugfix: MariaDbConnection extends MySqlConnection, so "MariaDB" is never shown currently.